.github/workflows/baseline-visual-regression.yml

```diff
@@ -0,0 +1,67 @@
+# This workflow runs Percy visual regression tests after a PR tagged with "package: skin"
+# is merged into the main branch. It extracts the list of stories from the PR body
+# and runs snapshots for those stories to update the visual regression baselines.
+
+name: Percy Merged Visual Regression Baselines
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  post-merge-snapshots:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checkout the repository
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      # Set up Node.js
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 22
+
+      # Install dependencies
+      - name: Install dependencies
+        run: npm install
+        working-directory: ./packages/skin
+
+      # Extract target stories from PR body
+      - name: Extract target stories
+        id: extract_stories
+        run: |
+          # Get the last merged PR details
+          PR_BODY=$(gh pr view $(git log -1 --pretty=format:"%s" | grep -oP '#\d+') --json body -q '.body')
```

Comment:
Isn't using the `env` property easier? I think this is the recommended way on the [github docs](https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable)
```
env:
  PR_BODY: ${{ github.event.pull_request.body}}
run:
  if [[ $PR_BODY ...
```

Reply:
Yep, changing...